ui/app/templates/vault/cluster/auth.hbs

```diff
@@ -6,27 +6,29 @@
   </Page.header>
   {{#if (has-feature "Namespaces")}}
     <Page.sub-header>
-      <div class="field is-horizontal">
-        <div class="field-label is-normal">
-          <label class="is-label" for="namespace">Namespace</label>
-        </div>
-        <div class="field-body">
-          <div class="field">
-            <div class="control">
-              <input
-                value={{namespaceQueryParam}}
-                placeholder="/ (Root)"
-                oninput={{perform updateNamespace value="target.value"}}
-                autocomplete="off"
-                name="namespace"
-                id="namespace"
-                class="input"
-                type="text"
-              />
+      <Toolbar @class="toolbar-namespace-picker">
```

Comment:
You shouldn't need `@` on `class` for angle bracket components (I know we had some because of an old polyfill we were using that needed it - should be fine to use w/o it though).